README.md

```diff
@@ -257,6 +257,12 @@ Any collection metadata mismatches will occur in a task with the type '`verifyCo
 
 In this case, '`failed_docs`' contains all the meta data mismatches, in this case an index named '`x_1`'.
 
+# Resumability
+
+The migration-verifier periodically persists its change stream’s resume token so that, in the event of a catastrophic failure (e.g., memory exhaustion), when restarted the verifier will receive any change events that happened while the verifier was down.
+
+**CAVEAT:** This doesn’t work properly during partitionining. If the verifier ends prior to partitioning, you should restart the verification.
```

Comment:
How does a user know whether it's prior or post partitioning?

Reply:
The last log line will indicate that it’s partitioning. I can clarify, though.

Reply:
I’m actually going to strike this paragraph since REP-5218 will fix partitioning.